docs/source/admin/log-messages.md

```diff
@@ -1,33 +1,30 @@
 # Interpreting common log messages
 
 When debugging errors and outages, looking at the logs emitted by
-JupyterHub is very helpful. This document tries to document some common
-log messages, and what they mean.
+JupyterHub is very helpful. This document intends to describe some common
+log messages, and what necessary actions to take.
 
-## Failing suspected API request to not-running server
+## Failing suspected API request to run-down server
 
 ### Example
 
-Your logs might be littered with lines that might look slightly scary
+Your log might display lines that seem cryptic
 
 ```
 [W 2022-03-10 17:25:19.774 JupyterHub base:1349] Failing suspected API request to not-running server: /hub/user/<user-name>/api/metrics/v1
 ```
 
 ### Most likely cause
 
-This likely means is that the user's server has stopped running but they
-still have a browser tab open. For example, you might have 3 tabs open, and shut
-your server down via one. Or you closed your laptop, your server was
-culled for inactivity, and then you reopen your laptop again! The
-client side code (JupyterLab, Classic Notebook, etc) does not know
-yet that the server is dead, and continues to make some API requests.
+The possible reason may be that the user's server has stopped running
+but they still have a browser tab open. For example, you might have 3 tabs open and you shut
+the server down via one. The other possible reason could be that you closed your laptop and the server was culled for inactivity, then reopen the laptop! However, the client-side code (JupyterLab, Classic Notebook, etc) doesn't interpret the shut-down server and continues to make some API requests.
```

Comment:
Let's put each main idea into it's own row to make it more easier to read and keep the same tense of verbs (`reopen` ➡️  `reopened`) 

```suggestion
the server down via one.
The other possible reason could be that you closed your laptop and the server was culled for inactivity, then reopened the laptop!
However, the client-side code (JupyterLab, Classic Notebook, etc) doesn't interpret the shut-down server and continues to make some API requests.
```